docs/platform/channels/sites.md

```diff
@@ -348,3 +348,115 @@ The precedence of locations on Modyo sites is as follows:
+are linked, with the `Referrer-Policy` header you can control how much information
+should be included in the Referer header.
+
+* **no-referrer**: No referrer information is sent.
+* **no-referrer-when-downgrade**: Don't send referrer information to a less
+  secure destination.
+* **origin**: Send the origin domain only and strip out the paths and query
+  string.
+* **origin-when-cross-origin**: Send referrer information for same origin requests
+  and strip out the paths and query string to other destinations.
+* **same-origin**: Send referrer information for same origin requests only.
+* **strict-origin**: Send the origin domain only for same security level
+  requests, and don't send referrer information to less secure destinations.
+* **strict-origin-when-cross-origin**: Send referrer information to same-origin
+requests. Send the origin only to same protocol security level and don't send
+  referrer information to less secure destinations.
+* **unsafe-url**: Always send referrer information.
+
+### X-Frame-Options
+
+Indicates if your site should be allowed to be rendered in a `frame`, `iframe`,
+`embed` or `object`.
+* **DENY**: The site cannot be displayed in a frame.
+* **SAMEORIGIN**: The site can be displayed on the same domain .
+
+### X-Content-Type-Options
+
+Indicates that advertised MIME types in the `Content-Type` header must be
+followed to prevent MIME type sniffing.
+
+### Content-Security-Policy
+
+Control what resources the browser is allowed to load for the site to
+mitigate cross site scripting and data injection attacks. The default value
+**allows to load resources from anywhere**, so it's important to design the right
+content cecurity Policy for your site.
+Specify your content security policy freely in text area, for a complete guide on how to
+write your policy see [Content Security Policy from MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP).
+
+:::warning Warning
+A very strict value can interfere with some features like
+[Google tag manager](/platform/channels/sites.html#google-tag-manager),
+[PWA](/platform/channels/sites.html#pwa),
+[Widgets](/platform/channels/widgets.html) and
+[Asset Manager](/platform/content/asset-manager.html).
+:::
+
+A ready for production policy must ensure that al the resources as
+images and stylesheets are loaded from trusted sources, and require that all scripts
+are safe an trusted by the application. For example, a strict policy for the
+minimal template would look like:
+
+```
+default-src 'self'; img-src 'self' https://cloud.modyocdn.com; font-src 'self' https://cloud.modyocdn.com http://cdn.materialdesignicons.com; style-src 'self' http://cdn.materialdesignicons.com; script-src 'self'
+```
+
+Your policy should include a `default-src 'self'` directive, which is a fallback
+for any other resource type. Also needs to include a `script-src` and a
+`style-src` directives to prevent inline `script` and `style` tags evaluation.
```

Comment:
```suggestion
for any other resource type. It also needs to include `script-src` and `style-src` directives to prevent evaluation of the `script` and `style` inline tags.
```